docs/docs-content/clusters/edge/local-ui/cluster-management/create-cluster.md

```diff
@@ -1,23 +1,25 @@
 ---
 sidebar_label: "Create Local Cluster"
 title: "Create Local Cluster"
-description: "Instructions for creating a locally manageg cluster in Edge Host Management Console."
+description: "Instructions for creating a locally managed cluster in Edge Host Management Console."
 hide_table_of_contents: false
-sidebar_position: 30
+sidebar_position: 20
 tags: ["edge"]
 ---
 
-You can create and manage a single-node cluster on an airgapped Edge host locally from Local UI. This page guides you
-through how to create a cluster using Local UI.
+You can create and manage single-node as well as multi-node clusters on airgapped hosts locally from Local UI. This page
+guides you through how to create a cluster using Local UI.
 
 :::preview
 
 :::
 
 ## Limitations
 
-- You can only create single-node clusters consisting solely of the Edge host you create the cluster from. You cannot
-  include other Edge hosts in the same local or remote network.
+- For multi-node clusters, all hosts must be deployed in the same deployment mode. For more information, refer to
+  [Deployment Modes](../../../../deployment-modes/deployment-modes.md).
+
+- For hosts that are deployed in agent mode, all hosts must share the same Operating System (OS).
 
 ## Prerequisites
```

Comment:
@lennessyy, for Airgap MNC clusters, should we add "stylus.enableMultiNode" flag set to "true" in user-data, as a prereq ?
Also, to upload content to MNC edge hosts - the CLI used to build airgap content should be at least 4.5.b. Content built from Pre-4.5.b CLI cannot be used in 4.5.b MNC Edge hosts.